### PR TITLE
Enhancement/update readme

### PR DESCRIPTION
**Pull request by SansoneG**
Suggestions welcome:

- Is anything missing for an easy quickstart?
- Is it consistent with [pilz_robots](https://github.com/PilzDE/pilz_robots/blob/melodic-devel/README.md)?

### CODE REVIEW

**Review by ct2034 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -90,15 +97,20 @@ To start reading from the Safety laser scanner and publishing complete scans exe
...
-roslaunch psen_scan psen_scan.launch host_ip:=192.168.0.20 host_udp_port:=3050
+roslaunch psen_scan psen_scan.launch sensor_ip:=192.168.0.10 host_ip:=192.168.0.20 host_udp_port:=3050
 ```
-This example configures the Safety laser scanner to send it´s frames to 192.168.0.20:3050.
+This example configures the Safety laser scanner at 192.168.0.10 to send it´s frames to 192.168.0.20:3050.
+
+### Adjust expert parameters
+If you've created an application package with your own launch file as described in the
+[tutorials](http://wiki.ros.org/psen_scan/Tutorials/),
+you can easily adjust many other configuration parameters.
```
Comment (jschleicher) [inline README.md]: many other: which ones exactly? This probably refers to the list of parameters in the above paragraph.
```suggestion
you can easily adjust the aforementioned configuration parameters.
```

**Review by jschleicher — COMMENTED** [review]
The tutorials link is hidden in a subsection "expert parameters"? How about adding a  prominent link to the wiki to the first paragraph as in [pilz_robots](https://github.com/PilzDE/pilz_robots/#package-pilz_robots)?

**Comment by SansoneG** [comment]
> The tutorials link is hidden in a subsection "expert parameters"? How about adding a  prominent link to the wiki to the first paragraph as in pilz_robots?

does 51aad81c68f7d8abb49c9636b692be484bc33ece satisfy?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
+[![PILZ](img/pilz-logo.png)](https://www.pilz.com)
+
 # PILZ Safety Laserscanner PSENscan
 
 ## Package: psen_scan
 
-The **psen_scan** package is a ROS integration of the PSENscan Safety laser scanner product. It allows for an easy integration of the Laserscanner-data into your ROS-Environment. Using the standard [sensor_msgs/LaserScan][] message format ensures compatibility with other laserscan-post-processing nodes such as [gmapping][].
+The **psen_scan** package is a ROS integration of the PSENscan Safety laser scanner product. It allows for an easy integration of the Laserscanner-data into your ROS-Environment. Using the standard [sensor_msgs/LaserScan][] message format ensures compatibility with other laserscan-post-processing nodes such as [gmapping][]. For a general overview and link collection we refer to the [wiki page](http://wiki.ros.org/psen_scan).
```
Comment (cdoehn) [inline README.md]:
```suggestion
The **psen_scan** package is a ROS integration of the PSENscan Safety laser scanner product. It allows for an easy integration of the laser scanner data into your ROS Environment. Using the standard [sensor_msgs/LaserScan][] message format ensures compatibility with other laserscan-post-processing nodes such as [gmapping][]. For a general overview and link collection we refer to the [wiki page](http://wiki.ros.org/psen_scan).
```

**Comment by cdoehn** [comment]
Consider to replace the first line by 

# PILZ Safety laser scanner PSENscan

File: README.md [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
+# PILZ Safety laser scanner PSENscan
 
 ## Package: psen_scan
 
-The **psen_scan** package is a ROS integration of the PSENscan Safety laser scanner product. It allows for an easy integration of the Laserscanner-data into your ROS-Environment. Using the standard [sensor_msgs/LaserScan][] message format ensures compatibility with other laserscan-post-processing nodes such as [gmapping][].
+The **psen_scan** package is a ROS integration of the PSENscan Safety laser scanner product. It allows for an easy integration of the laser scanner data into your ROS Environment. Using the standard [sensor_msgs/LaserScan][] message format ensures compatibility with other laserscan-post-processing nodes such as [gmapping][]. For a general overview and link collection we refer to the [wiki page](http://wiki.ros.org/psen_scan).
 
 <p align="center">
-<img src="img/PSENscan.jpg" alt="Pilz Safety Laserscanner" title="Pilz Safety Laserscanner">
+<img src="img/PSENscan.jpg" alt="Pilz Safety laser scanner" title="Pilz Safety laser scanner">
```
Comment (jschleicher) [inline README.md]: maybe adjust to uppercase from heading?
```suggestion
<img src="img/PSENscan.jpg" alt="PILZ Safety laser scanner" title="PILZ Safety laser scanner">
```

File: README.md [hunk 4]
```diff
@@ -90,15 +97,20 @@ To start reading from the Safety laser scanner and publishing complete scans exe
...
+If you've created an application package with your own launch file as described in the
+[tutorials](http://wiki.ros.org/psen_scan/Tutorials/),
+you can easily adjust the aforementioned configuration parameters.
 
 ## You need further information?
 Our international hotline staff will support you individually about our ROS packages at
 ros@pilz.de
 
-Find more information about the Pilz manipulator module on the [product website](https://www.pilz.com/en-INT/eshop/00106002197131/PSENscan-Safety-Laser-Scanner).
+Find more information about the Pilz Safety laser scanner on the [product website](https://www.pilz.com/en-INT/eshop/00106002197131/PSENscan-Safety-Laser-Scanner).
```
Comment (jschleicher) [inline README.md]: Maybe use lower-case? Or same as in headline?
```suggestion
Find more information about the Pilz safety laser scanner on the [product website](https://www.pilz.com/en-INT/eshop/00106002197131/PSENscan-Safety-Laser-Scanner).
```

**Review by jschleicher — APPROVED** [review]
great!

See tiny spelling-nitpicks inline.

File: README.md (hunk 3 above)
Comment (SansoneG) [reply README.md]: Solved in 01dcea65f79ebae8a82975a9271cb7ec29a87550

File: README.md (hunk 4 above)
Comment (SansoneG) [reply README.md]: Solved in 01dcea65f79ebae8a82975a9271cb7ec29a87550

**Pull request merged**